server/api/fileOperations.js

```diff
@@ -59,4 +61,24 @@ router.post("fileOperations.list", auth(), pagination(), async (ctx) => {
   };
 });
 
+router.post("fileOperations.redirect", auth(), async (ctx) => {
+  const { id } = ctx.body;
+  ctx.assertPresent(id, "id is required");
+
+  const user = ctx.state.user;
+  const team = await Team.findByPk(user.teamId);
+
+  const fileOp = await FileOperation.findByPk(id);
+  if (!fileOp) {
+    throw new NotFoundError();
+  }
+
+  authorize(user, fileOp.type, team);
+
+  const accessUrl = await getSignedUrl(fileOp.key);
```

Comment:
Now that we are using `key` to access the URL. I think we should remove the `url` attribute from the model. What do you think?

Reply:
Agreed to remove from public responses